spec/plans/results.fmf

```diff
@@ -33,6 +33,12 @@ description: |
          some-id: foo
          another-id: bar
 
+       # String, when the test started, in an ISO 8601 format.
+       starttime: "yyyy-mm-ddThh:mm:ss.mmmmm+ZZ:ZZ"
```

Comment:
So far we haven't been using camel case or multiword for any key names (except for `serialnumber` which slipped through the cracks and should be fixed IMO). Can we keep the naming consistent (e.g. always use dashes for separating multiword options and yaml keys and underscores when the key name is to be used in python code)?

Reply:
Easy to propose, harder to achieve :/ I agree, but I ran into soo many issues with `serial-number` that I gave up in favor of getting it to work sooner. How about merging this PR first, then I'd fix the naming of all three keys?

Reply:
Filed #2054 to track this.